src/phase0/beacon_state.rs

```diff
@@ -61,3 +70,170 @@ pub struct BeaconState<
     pub current_justified_checkpoint: Checkpoint,
     pub finalized_checkpoint: Checkpoint,
 }
+
+pub fn get_current_epoch<
```

Comment:
and if you consolidate/move the accessors into the `state_transition` module you won't duplicate the `get_current_epoch` function

Reply:
fixed